.github/workflows/elixir.yml

```diff
@@ -18,13 +18,16 @@ jobs:
           - "1.16"
           - "1.17"
           - "1.18"
+          - "1.19"
         otp:
           - "25"
           - "26"
           - "27"
           - "28"
         # see https://hexdocs.pm/elixir/compatibility-and-deprecations.html#between-elixir-and-erlang-otp
         exclude:
+          - elixir: 1.19
+            otp: 25
```

Comment:
Maybe this should be 28?

Reply:
This is the exclusion list.  

Why would be want to exclue 1.19 on OTP 28 when it's support from 26-28? https://hexdocs.pm/elixir/1.19.0-rc.0/compatibility-and-deprecations.html#between-elixir-and-erlang-otp

Reply:
Oh! Sorry, I just misread that